Removing hard coded place holders in templates as the build pipelines doesnt handle it well

eladiw opened the pull request:
### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Microsoft/agogosml/blob/master/CONTRIBUTING.rst) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Microsoft/agogosml/pulls) for the same update/change?
- [ ] Does your PR follow our [Code of Conduct](https://opensource.microsoft.com/codeofconduct/)?

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Does each method or function "do one thing well"? Reviewers may recommend methods be split up for maintainability and testability.
- [ ] Is this code designed to be testable? 
- [ ] Is the code documented well?
- [ ] Does your submission pass existing tests (or update existing tests with documentation regarding the change)?
- [ ] Have you added tests to cover your changes?
- [ ] Have you linted your code prior to submission?
- [ ] Have you updated the documentation and README?
- [ ] Is PII treated correctly? In particular, make sure the code is not logging objects or strings that might contain PII (e.g. request headers). 
- [ ] Have secrets been stripped before committing?

sayar commented:
@eladiw Can you add documentation somewhere about the values that need to be filled (given that the placeholders are removed now)?

eladiw commented:
> @eladiw Can you add documentation somewhere about the values that need to be filled (given that the placeholders are removed now)?

@sayar thanks. I have added a simple readme file to elaborate and explain the process and missing arguments. also one of the missing values was possible to turn into an argument, so I fixed it
